views/TrustedBy.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const data = [
+	{ logo: '/ablejobs.svg', heading: 'Ablejobs'},
+	{ logo: '/blance.png', heading: 'Blance'},
+	{ logo: '/supplynote.svg', heading: 'SupplyNote'},
```

Comment:
Doesn't this break for you? I don't see these images added to the repository. Let me know if you want me to add them.

Reply:
Done